jwst/outlier_detection/outlier_detection_scaled.py

```diff
@@ -84,7 +84,12 @@ def do_detection(self):
             radius_outer = 5
 
         apertures = CircularAperture((xcenter, ycenter), r=radius)
-        aperture_mask = apertures.to_mask(method='center')[0]
+        # ApertureMask in photutils<=0.6 is indexable.  In >=0.6 it is scalar
```

Comment:
Title of the PR says ">=0.7", while this says ">=0.6". So which is it?

Reply:
PR title is correct.  Will change the comment.